frontend/src/services/EnvService.ts

```diff
@@ -0,0 +1,20 @@
+import { UiCookieValues } from '~shared/types'
+
+import { PUBLIC_FORMS_ENDPOINT } from '~features/public-form/PublicFormService'
+
+import { ApiService } from './ApiService'
+
+const ENV_ENDPOINT = '/environment'
+const ADMIN_ENDPOINT = '/admin'
+
+export const publicChooseEnvironment = async (): Promise<UiCookieValues> => {
```

Comment:
do you think we should create an issue and link a TODO here so we remember to remove all these related code when we go 100% React?

(and everywhere else that is related to these functions)

Reply:
good idea, will reference in TODOs + have created a [new issue](https://github.com/opengovsg/FormSG/issues/4279) to tag it to